go.mod

```diff
@@ -3,20 +3,20 @@ module storj.io/utp-go
 go 1.18
 
 require (
-	github.com/go-logr/logr v0.4.0
-	github.com/go-logr/zapr v0.4.0
-	github.com/stretchr/testify v1.4.0
-	go.uber.org/zap v1.16.0
+	github.com/go-logr/logr v1.2.4
+	github.com/go-logr/zapr v1.2.3
```

Comment:
I wonder whether we should slowly start to migrate towards using slog?

Also, currently uplink library has been written such that it doesn't bring in zap. For whatever reason zap depends on net/http, which is unnecessary.